tests/test-rest-posts-controller.php

```diff
@@ -482,6 +482,7 @@ public function test_get_items_valid_tax_query_in_filter() {
 
 		$request = new WP_REST_Request( 'GET', '/wp/v2/posts' );
 		$request->set_param( 'filter', array(
+			// WPCS: tax_query ok.
 			'tax_query' => array(
```

Comment:
You need to add the comments inline.  So this would be the correct syntax `'tax_query' => array( // WPCS: tax_query ok.` or at least I think :)